lib/config-mac.h

```diff
@@ -53,6 +53,7 @@
 #define HAVE_UTIME_H            1
 #define HAVE_SYS_TIME_H         1
 #define HAVE_SYS_UTIME_H        1
+#define HAVE_SYS_FILIO_H        1
```

Comment:
Daft question since I don't know Mac OS other than as a user (a long time ago..). Is `filio.h` available on all versions of Mac OS on which it would be feasible to build libcurl? In other words, does it go further back than OS9?

Disclaimer: I'm mainly curious, I don't think it's a problem to define this if it doesn't

Reply:
Classic Mac OS does not have filio.h or other Unix headers. To build curl on classic Mac OS, the [Grand Unified Socket Interface (GUSI)](https://sourceforge.net/projects/gusi/) is required. It provides an early 2000s Unix-like socket and threads interface for classic Mac OS. It is compatible with System 7, Mac OS 8, and Mac OS 9.

Reply:
Gotcha, thanks.